RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -8,21 +8,21 @@
     applyPulseJitter      = cms.bool(False),
     applyTimeSlew         = cms.bool(True),           #units
     ts4Min                = cms.double(0.),           #fC
-    ts4Max                = cms.vdouble(100.,70000.), #fC # this is roughly 20 GeV
+    ts4Max                = cms.vdouble(100.,45000.), #fC # this is roughly 20 GeV
```

Comment:
what kind of variation can we expect here?
What was it for HPDs?
Is it o(10%) or O(100%)?